extensions/amp-story/amp-story.md

```diff
@@ -338,6 +338,18 @@ A URL to the [story poster](#posters) in square format (1x1 aspect ratio).
 
 A URL to the [story poster](#posters) in landscape format (4x3 aspect ratio).
 
+##### live-story [optional]
+
+Enables the [live stories](#Live-stories) functionality.
+
+##### live-story-disabled [optional]
+
+Stops polling the server when using the live-story attribute. Use this when you want to stop sending live updates to the story.
```

Comment:
The word "server" makes it confusing, as a developer it'd be hard to know what that means: is it an endpoint just like `amp-access` or `amp-story-bookend`?
The `amp-live-list` documentation uses the term "origin document on the host" instead.
There's more information that'd be useful to publishers, ie: _"if the user is viewing a document served from an AMP cache, the client will poll that document hosted on that AMP cache for updates; if the user is viewing a document served from a web publisher's origin domain (e.g. "example.com"), then the client will poll the document hosted on that origin domain for updates."_
Should we just rely on the `amp-live-list` documentation, by linking to it somewhere for interested publishers?

"sending live updates to the story": could we specify more accurately what a live update is? My point is that to someone with 0 context, it could mean anything from real time notifications (push) to live video streaming. The only use case as of today is adding new `amp-story-page` elements to the story, with no update/deletion available. Could you reflect that too somewhere?

Reply:
That's a good point. I updated the description on the attributes to make it as simple as possible, elaborating more on the actual `Live Story` section. 

I copied most of the relevant text coming from [amp-live-list](https://amp.dev/documentation/components/amp-live-list) and adapted it to the story format. I prefer not to link to the amp-live-list documentation since it would be overhead for publishers to learn about it, and since we're not supporting all of the attributes that `amp-live-list` offers, it's better for them not to try and use them.

I will send a follow-up to disallow `data-tombstone` and `data-update-time` in the validator.